site/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="gemi-nikk">
+    <meta name="description" content="A personal website featuring a bio, favorite things, a haiku, and useful links about .">
+    <title>kishan's CSD-112 Project</title>
+    <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <link rel="shortcut icon" href="/favicon.ico" />
```

Comment:
1. `Lab 2 Site has > root with > a favicon`
```suggestion
    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
```